src/Compilers/CSharp/Portable/Symbols/Source/CustomModifierUtils.cs

```diff
@@ -35,8 +35,7 @@ internal static class CustomModifierUtils
             // have already been compared.
             MethodSymbol constructedSourceMethod = sourceMethod.ConstructIfGeneric(destinationMethod.TypeArguments);
 
-            customModifiers = CustomModifiersTuple.Create(constructedSourceMethod.ReturnTypeCustomModifiers,
-                                                          destinationMethod.ReturnsByRef ? constructedSourceMethod.RefCustomModifiers : ImmutableArray<CustomModifier>.Empty);
+            customModifiers = CustomModifiersTuple.Create(constructedSourceMethod.ReturnTypeCustomModifiers, constructedSourceMethod.RefCustomModifiers);
```

Comment:
This was marked as resolved, but I didn't notice the tests. The scenario would be when overriding method is not ref returning, but the overridden method is ref readonly returning.

---
In reply to: [118421677](https://github.com/dotnet/roslyn/pull/19658#discussion_r118421677) [](ancestors = 118421677)